CRAYSAT-2032: Fix `NoneType` object error from get_ssh_client (#371)
Cray-HPE/sat: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [3.36.1] - 2025-09-02

### Fixed

> - Skip unparsable known_hosts entries in get_ssh_client to prevent
>   "`NoneType` object has no attribute `hostnames` from get_ssh_client"

Reviewer: Minor improvement suggestion:

```suggestion
- Skip unparsable known_hosts entries in `get_ssh_client` to prevent a traceback
  with message "AttributeError: `NoneType` object has no attribute `hostnames`"
```
